packages/graphql-transformers/src/openCrudWhereTransform/openCRUDtoSequelizeWhereTransformer.ts

```diff
@@ -104,8 +106,9 @@ function openCrudFilterToSequelize(whereArg, whereValue, entityName, pathWithinS
     sqIncludeElement = flatMap(sqElements, 'include');
   } else {
     const whereArgField = getField(openCrudSchema, entityName, whereArg) as any;
-    if (isScalar(whereArgField.type)) {
-      sqWhereElement = handleScalarField(whereArgField, whereArg, whereValue, entityName, useColumnNames);
+    const isArrayField = isList(whereArgField.type) && isArrayFilter(whereArg);
+    if (isScalar(whereArgField.type) || isArrayField) {
+      sqWhereElement = handleScalarField(whereArgField, whereArg, whereValue, entityName, useColumnNames, isArrayField);
```

Comment:
It's a bit confusing hat we're using the same handler of handleScalarField for non-scalar fields, e.g Arrays.

Reply:
I agree that may be confusing.

Possible alternative name could be `handleBaseTypeField`,
as per [Postgres docs](https://www.postgresql.org/docs/9.6/extend-type-system.html):

> Base types are further subdivided into scalar and array types. 

@SegevHaviv what's your view on such name?